app/database/models/personal_background.py

```diff
@@ -0,0 +1,146 @@
+from sqlalchemy import null
+from app.database.sqlalchemy_extension import db
+from app.database.db_types.JsonCustomType import JsonCustomType
+from app.utils.bitschema_utils import (
+    Gender,
+    Age,
+    Ethnicity,
+    SexualOrientation,
+    Religion,
+    PhysicalAbility,
+    MentalAbility,
+    SocioEconomic,
+    HighestEducation,
+    YearsOfExperience,
+)
+
+
+class PersonalBackgroundModel(db.Model):
+    """Defines attributes for user's personal background.
+
+    Attributes:
+    user_id: An integer for storing the user's id.
+    gender: A string for storing the user's gender.
+    age: A string for storing the user's age.
+    ethnicity: A string for storing the user's wthnicity.
+    sexual_orientation: A string for storing the user's sexual orientation.
+    religion: A string for storing the user's religion.
+    physical_ability: A string for storing the user's physical ability.
+    mental_ability: A string for storing the user's mental ability.
+    socio_economic: A string for storing the user's socio economic level.
+    highest_education: A string for storing the user's highest education level.
+    years_of_experience: A string for storing the user's length of expeprience in the It related area.
+    others: A JSON data type for storing users descriptions of 'other' fields.
+    is_public: A boolean indicating if user has agreed to display their personal background information publicly to other members.
+    """
+
+    # Specifying database table used for PersonalBackgroundModel
+    __tablename__ = "personal_backgrounds"
+    __table_args__ = {"extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # User's personal background data
+    user_id = db.Column(
+        db.Integer,
+        db.ForeignKey("users.id", ondelete="CASCADE"),
+        nullable=False,
+        unique=True,
+    )
+    gender = db.Column(db.Enum(Gender))
+    age = db.Column(db.Enum(Age))
+    ethnicity = db.Column(db.Enum(Ethnicity))
+    sexual_orientation = db.Column(db.Enum(SexualOrientation))
+    religion = db.Column(db.Enum(Religion))
+    physical_ability = db.Column(db.Enum(PhysicalAbility))
+    mental_ability = db.Column(db.Enum(MentalAbility))
+    socio_economic = db.Column(db.Enum(SocioEconomic))
+    highest_education = db.Column(db.Enum(HighestEducation))
+    years_of_experience = db.Column(db.Enum(YearsOfExperience))
+    others = db.Column(JsonCustomType)
+    is_public = db.Column(db.Boolean)
+
```

Comment:
@epicadk , do you mean years of experience? First of all, I believe this is relevant to support one of BIT goals which is to improve equity in IT because many with no to limited years of experience in IT were left out from employment opportunity. We often see a position for entry level required more than 3 - 5 years of experience. Stating this years of experience encouraging industries to target people within the lower years of experience to be the mentee and people with the higher level of experience to be the mentor. Their willingness to go against the norm in employment market (aka to be more inclusive by targeting those within the minority groups, e.g. minimal years of experience) is going to be highlighted in the BIT Landing page as per the Mockup in my GSoC20 proposal.
![image](https://user-images.githubusercontent.com/29667122/117105558-a399ef80-adc1-11eb-9566-175803f050d5.png)